vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -3725,6 +3727,7 @@ quantitykind:Distance
 .
 quantitykind:DistanceTraveledDuringBurn
   a qudt:QuantityKind ;
+  qudt:hasDimensionVector qkdv:A0E0L1I0M0H0T0D0 ;
   qudt:symbol "s" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Distance Traveled During a Burn"@en ;
```

Comment:
What is the context for this? My colleagues who work with wildfires might be interested? Or is it drag-racing? or rocketry? 

Implication: the label and description might need to be more explicit.

Reply:
It is indeed rocketry, from the NASA days. There are a number of these very rocketry-specific quantity kinds. I'm not really convinced they should stay, but they aren't doing any harm...